client/extensions/wp-job-manager/components/settings/job-listings/index.jsx

```diff
@@ -274,7 +274,7 @@ class JobListings extends Component {
 
 const connectComponent = connect(
 	( state ) => {
-		const selector = formValueSelector( 'jobListings', () => state.extensions.wpJobManager.form );
+		const selector = formValueSelector( 'wpJobManager.jobListings', () => state.ui.form );
```

Comment:
A little performance improvement can be done here: move the definition of the selector outside the`connect` call -- create it once when initializing the module and avoid recreating it on every state update:
```js
const selector = formValueSelector( 'wpJobManager.jobListings', state => state.ui.form );
const connectComponent = connect( state => ( {
  perPage: selector( state, 'listings.perPage' )
} ) );
```